instrumentation/graphql-java-16.2/src/test/java/com/nr/instrumentation/graphql/GraphQLObfuscatorTest.java

```diff
@@ -0,0 +1,28 @@
+package com.nr.instrumentation.graphql;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvFileSource;
+
+import static com.nr.instrumentation.graphql.helper.GraphQLTestHelper.readText;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class GraphQLObfuscatorTest {
+
+    private final static String OBFUSCATE_DATA_DIR = "obfuscateQueryTestData";
+
+    @ParameterizedTest
+    @CsvFileSource(resources = "/obfuscateQueryTestData/obfuscate-query-test-data.csv", delimiter = '|', numLinesToSkip = 2)
+    public void testObfuscateQuery(String queryToObfuscateFile, String expectedObfuscatedQueryFile) {
+        //setup
+        queryToObfuscateFile = queryToObfuscateFile.trim();
+        expectedObfuscatedQueryFile = expectedObfuscatedQueryFile.trim();
+        String expectedObfuscatedResult = readText(OBFUSCATE_DATA_DIR, expectedObfuscatedQueryFile);//readText(expectedObfuscatedQueryFile);
```

Comment:
Can this commented out code (`//readText(expectedObfuscatedQueryFile);`) be removed?

Reply:
Yes, definitely...good catch